app/views/families/index.html.erb

```diff
@@ -21,19 +21,19 @@
       <table class="table table-striped">
         <thead>
         <tr>
-          <th scope="col">Guardian first name</th>
           <th scope="col">Guardian last name</th>
+          <th scope="col">Guardian first name</th>
           <th scope="col">Agency guardian</th>
           <th scope="col">Comments</th>
           <th scope="col"></th>
           <th scope="col"></th>
         </tr>
         </thead>
         <tbody>
-        <% families.each do |family| %>
+        <% Family.order(:guardian_last_name).each do |family| %>
```

Comment:
This is one of the dowsides to the way we use helper methods vs instance variables (something Ive been meaning to change) This should actually be on `families` and not on the class `Family`, otherwise we risk exposing data between partners.

Reply:
I think this still needs to be addressed?

Reply:
Ack! Good catch. Fixed now.